static/src/javascripts/projects/common/views/acquisitions-epic-testimonials-prominent.html

```diff
@@ -0,0 +1,41 @@
+<div class="contributions__epic" data-component="<%=componentName%>">
+    <div>
+        <h2 class="contributions__title contributions__title--epic">
+            Why our readers are supporting us:
+        </h2>
+        <div class="epic__testimonial__container">
+            <p class="epic__testimonial__text">
+                I read it fairly often and enjoy the variety of news and entertainment content.
+                It never occurred to me that you wanted contributions, but I'd like you to stick around.
+                Sometimes you actually have to pay for stuff you like. Go figure.
+            </p>
+            <p class="epic__testimonial__name">
```

Comment:
Once the parent is converted, this could be the `<cite />` element then (which lives inside of `blockquote`). This could be helpful: http://html5doctor.com/blockquote-q-cite/